lib/index.js

```diff
@@ -36,5 +52,15 @@ Perimeterx.prototype.initialize = function() {
 Perimeterx.prototype.loaded = function() {
   // what are required properties or functions that you need available on the `window`
   // before the integration marks itself as ready?
+  window._pxAppId = this.options.appId;
+
+  // iterate over customTraits and set is as pxParam
+  // window._pxParam1 = "<param1>";
+  var traits = this.options.customTraits;
```

Comment:
I think we need to revisit how custom traits should be sent. Will follow up in the email thread